microsoft/vscode
# Add output channel for telemetry.log file

This PR adds a output channel for the `telemetry.log` file and addresses the issue #58536

@sandy081 When opening with `--verbose` flag, the telemetry events get logged to the `telemetry.log` file right from the start. But the output channel remains blank until I close it and open it again. Any ideas why so?

Note: To test this locally, 
- remove the `isBuilt` check at https://github.com/Microsoft/vscode/blob/74b096b6be864f92cdeadae2304b7d872034669f/src/vs/code/electron-browser/sharedProcess/sharedProcessMain.ts#L106 
- enable telemetry in product.json, provide any dummy value for aiConfig.asimovKey
